src/styles/footer.scss

```diff
@@ -0,0 +1,89 @@
+.footer {
+    padding: 5em 0 5em 0;
+    display: flex;
+    overflow: hidden;
+    flex-direction: column;
+    align-items: center;
+    text-align: center;
+
+    &__socials {
+        .mobile-icons {
+            display: none;
+        }
+        a {
+        justify-content: space-between;
+        align-items: center;
+        padding: 0 0.5rem;
+        margin-left: auto;
+        margin-right: auto;
+        color: $dark-blue;
+    }}
+
+
+    svg {
+        width: 150px;
+        height: 150px;
+    }
+}
+
+input {
+    border: 0;
+    outline: 0;
+    width: 300px;
+    height: 40px;
+    box-sizing: border-box;
+}
+
+.footer__form {
+    width: 100%;
+    padding: 1em 0;
+
+    &-heading {
```

Comment:
for styling for .scss we have been using __ so something like `&__heading` just to stay consistent :)